server/testfixtures/test-repos/workspace-parallel-yaml/exp-output-apply-all-staging.txt

```diff
@@ -32,3 +32,4 @@ workspace = "staging"
 
 ---
 
+,
```

Comment:
Why is there a trailing comma though? Is this a bug?

Reply:
I feel like this is bug, that is the fix for the broken tests.